dicoogle/src/main/resources/webapp/js/components/indexer/IndexStatusView.js

```diff
@@ -81,7 +81,10 @@ const IndexStatusView = createReactClass({
     if (this.state.data.tasks.length === 0) {
       items = <div>No tasks</div>;
     } else {
-      items = this.state.data.tasks.map(item => (
+      items = this.state.data.tasks.sort((item1, item2) => {
+        // Sorts tasks from latest to first
+        return item2.taskTimeCreated - item1.taskTimeCreated;
```

Comment:
```suggestion
        return new Date(item2.taskTimeCreated).getTime() - new Date(item1.taskTimeCreated).getTime();
```

Reply:
Changed according to attribute format change.